### ISSUE TITLE
Wildcard [Charts]: Improve contrast of active/non-active colours 

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#40970]
Follow up for https://github.com/sourcegraph/sourcegraph/pull/40962#discussion_r956933629

@valerybugakov noted that the focused/hovered active bar doesn't have enough contrast to be a11y compliant/ it also seems like a bug visually. 

In this issue, we need to improve our chart colour algorithm to avoid low contrast between active and non-active bars on the chart.
